docs/wdc_upgrade.md

```diff
@@ -3,7 +3,7 @@ title: Upgrading from WDC Version 1.x
 layout: docs
 ---
 
-If you have built connectors using WDC version 1.x you can continue to use those connectors in future versions of Tableau. However, if you link a version 1.x connector to version 2.x of the Tableau WDC library, the connector will not work unless you also update the connector to use code changes introduced in version 2.x of the API.
+If you have built connectors using WDC version 1.x you can continue to use those connectors in future versions of Tableau (see [WDC Versions]({{ site.baseurl }}\docs\wdc_library_versions)). However, if you link a version 1.x connector to version 2.x of the Tableau WDC library, the connector will not work unless you also update the connector to use code changes introduced in version 2.x of the API.
```

Comment:
I don't think the second sentence is very clear -- "if you like a version 1 connector to version 2 wdclib...".

Reply:
Wait what, 1.x will work? This section confuses me.

Reply:
@Kovner We want to developers know that they can't turn their version 1 WDC into a version 2 WDC by just pointing to a version 2.x library.  

I'll try rewording that.

Reply:
@Kovner  OK. I changed it again and cleared up what we meant by future versions of Tableau.  

If you have built connectors using WDC version 1.x you can continue to use those connectors in Tableau, up through Tableau 2019.1. See [WDC Versions]({{ site.baseurl }}\docs\wdc_library_versions) for information about compatibility. However, you cannot turn a version 1.x connector into a version 2.x connector by just linking to version 2.x of the Tableau WDC library. The connector will not work unless you also update the connector to use code changes introduced in version 2.x of the API.